website_wow/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/license-GPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/gpl-3.0-standalone.html
+   :alt: License: GPL-3
+
+===========
+Website Wow
+===========
+
+Provides animations for your Odoo website using WowJS, which is powered by `animate.css`.
```

Comment:
I was actually going for italics on both of these, because they're titles of things and not really code. IMO code blocks for things like this detract from the statement that is being made due to how much different they are from the text.